metals/src/main/scala/scala/meta/internal/rename/RenameProvider.scala

```diff
@@ -45,9 +46,11 @@ final class RenameProvider(
     workspace: AbsolutePath,
     client: MetalsLanguageClient,
     buffers: Buffers,
-    compilations: Compilations
+    compilations: Compilations,
+    metalsConfig: MetalsServerConfig
 ) {
 
+  private val renameFileTreshold = 100
```

Comment:
I don't think this is used anywhere is it since we are taking the threshold from the `metalsConfig`?

Reply:
Good catch, I forgot to remove that!

Reply:
Fixed